src/lib/core/error/error-options.ts

```diff
@@ -24,3 +25,31 @@ export class ErrorStateMatcher {
     return !!(control && control.invalid && (control.touched || (form && form.submitted)));
   }
 }
+
+
+export abstract class HasErrorStateComponent {
```

Comment:
Add a description for this class and how it's meant to be used?

Reply:
Deleted.